
 backend-2 Pulling 
 backend-1 Pulling 
 0368fd46e3c6 Already exists 
 0670596c9dd1 Already exists 
 b2683b5bf4e2 Already exists 
 a9d6291cfc79 Pulling fs layer 
 cd4bdbfa1a71 Pulling fs layer 
 0368fd46e3c6 Already exists 
 0670596c9dd1 Already exists 
 b2683b5bf4e2 Already exists 
 a9d6291cfc79 Pulling fs layer 
 cd4bdbfa1a71 Pulling fs layer 
 a9d6291cfc79 Verifying Checksum 
 a9d6291cfc79 Download complete 
 a9d6291cfc79 Extracting [==================================================>]      93B/93B
 a9d6291cfc79 Downloading [==================================================>]      93B/93B
 a9d6291cfc79 Verifying Checksum 
 a9d6291cfc79 Download complete 
 a9d6291cfc79 Extracting [==================================================>]      93B/93B
 a9d6291cfc79 Extracting [==================================================>]      93B/93B
 a9d6291cfc79 Extracting [==================================================>]      93B/93B
 a9d6291cfc79 Pull complete 
 a9d6291cfc79 Pull complete 
 cd4bdbfa1a71 Downloading [>                                                  ]  211.8kB/20.63MB
 cd4bdbfa1a71 Downloading [>                                                  ]  211.8kB/20.63MB
 cd4bdbfa1a71 Downloading [==================>                                ]  7.455MB/20.63MB
 cd4bdbfa1a71 Downloading [==================>                                ]  7.455MB/20.63MB
 cd4bdbfa1a71 Downloading [==============================>                    ]  12.79MB/20.63MB
 cd4bdbfa1a71 Downloading [==============================>                    ]  12.79MB/20.63MB
 cd4bdbfa1a71 Downloading [=============================================>     ]  18.68MB/20.63MB
 cd4bdbfa1a71 Downloading [=============================================>     ]  18.68MB/20.63MB
 cd4bdbfa1a71 Verifying Checksum 
 cd4bdbfa1a71 Download complete 
 cd4bdbfa1a71 Verifying Checksum 
 cd4bdbfa1a71 Download complete 
 cd4bdbfa1a71 Extracting [>                                                  ]  229.4kB/20.63MB
 cd4bdbfa1a71 Extracting [>                                                  ]  229.4kB/20.63MB
 cd4bdbfa1a71 Extracting [====================>                              ]  8.258MB/20.63MB
 cd4bdbfa1a71 Extracting [====================>                              ]  8.258MB/20.63MB
 cd4bdbfa1a71 Extracting [===========================================>       ]  17.89MB/20.63MB
 cd4bdbfa1a71 Extracting [===========================================>       ]  17.89MB/20.63MB
 cd4bdbfa1a71 Extracting [===============================================>   ]  19.73MB/20.63MB
 cd4bdbfa1a71 Extracting [===============================================>   ]  19.73MB/20.63MB
 cd4bdbfa1a71 Extracting [==================================================>]  20.63MB/20.63MB
 cd4bdbfa1a71 Extracting [==================================================>]  20.63MB/20.63MB
 cd4bdbfa1a71 Extracting [==================================================>]  20.63MB/20.63MB
 cd4bdbfa1a71 Extracting [==================================================>]  20.63MB/20.63MB
 cd4bdbfa1a71 Pull complete 
 cd4bdbfa1a71 Pull complete 
 backend-2 Pulled 
 backend-1 Pulled 
 Network micheloliveira-com-dotnet-csharp-redis_backend  Creating
 Network micheloliveira-com-dotnet-csharp-redis_backend  Created
 Container micheloliveira-com-dotnet-csharp-redis-redis-1  Creating
 Container micheloliveira-com-dotnet-csharp-redis-backend-1-1  Creating
 Container micheloliveira-com-dotnet-csharp-redis-backend-2-1  Creating
 Container micheloliveira-com-dotnet-csharp-redis-backend-2-1  Created
 Container micheloliveira-com-dotnet-csharp-redis-backend-1-1  Created
 Container micheloliveira-com-dotnet-csharp-redis-load-balancer-1  Creating
 Container micheloliveira-com-dotnet-csharp-redis-redis-1  Created
 Container micheloliveira-com-dotnet-csharp-redis-load-balancer-1  Created
Attaching to backend-1-1, backend-2-1, load-balancer-1, redis-1
backend-2-1      | [Redis] Attempting connection...
backend-2-1      | Retry 1: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
backend-1-1      | [Redis] Attempting connection...
backend-1-1      | Retry 1: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
redis-1          | Starting Redis Server
redis-1          | 1:C 30 Jul 2025 01:46:15.702 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
redis-1          | 1:C 30 Jul 2025 01:46:15.702 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis-1          | 1:C 30 Jul 2025 01:46:15.703 * Redis version=8.0.3, bits=64, commit=00000000, modified=1, pid=1, just started
redis-1          | 1:C 30 Jul 2025 01:46:15.703 * Configuration loaded
redis-1          | 1:M 30 Jul 2025 01:46:15.703 * monotonic clock: POSIX clock_gettime
redis-1          | 1:M 30 Jul 2025 01:46:15.703 * Running mode=standalone, port=6379.
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> RedisBloom version 8.0.1 (Git=unknown)
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> Registering configuration options: [
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> 	{ bf-error-rate       :      0.01 }
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> 	{ bf-initial-size     :       100 }
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> 	{ bf-expansion-factor :         2 }
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> 	{ cf-bucket-size      :         2 }
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> 	{ cf-initial-size     :      1024 }
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> 	{ cf-max-iterations   :        20 }
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> 	{ cf-expansion-factor :         1 }
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> 	{ cf-max-expansions   :        32 }
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * <bf> ]
redis-1          | 1:M 30 Jul 2025 01:46:15.704 * Module 'bf' loaded from /usr/local/lib/redis/modules//redisbloom.so
redis-1          | 1:M 30 Jul 2025 01:46:15.707 * <search> Redis version found by RedisSearch : 8.0.3 - oss
redis-1          | 1:M 30 Jul 2025 01:46:15.707 * <search> RediSearch version 8.0.1 (Git=5688fcc)
redis-1          | 1:M 30 Jul 2025 01:46:15.707 * <search> Low level api version 1 initialized successfully
redis-1          | 1:M 30 Jul 2025 01:46:15.707 * <search> gc: ON, prefix min length: 2, min word length to stem: 4, prefix max expansions: 200, query timeout (ms): 500, timeout policy: return, cursor read size: 1000, cursor max idle (ms): 300000, max doctable size: 1000000, max number of search results:  1000000, 
redis-1          | 1:M 30 Jul 2025 01:46:15.707 * <search> Initialized thread pools!
redis-1          | 1:M 30 Jul 2025 01:46:15.707 * <search> Disabled workers threadpool of size 0
redis-1          | 1:M 30 Jul 2025 01:46:15.707 * <search> Subscribe to config changes
redis-1          | 1:M 30 Jul 2025 01:46:15.707 * <search> Enabled role change notification
redis-1          | 1:M 30 Jul 2025 01:46:15.708 * <search> Cluster configuration: AUTO partitions, type: 0, coordinator timeout: 0ms
redis-1          | 1:M 30 Jul 2025 01:46:15.708 * <search> Register write commands
redis-1          | 1:M 30 Jul 2025 01:46:15.708 * Module 'search' loaded from /usr/local/lib/redis/modules//redisearch.so
redis-1          | 1:M 30 Jul 2025 01:46:15.708 * <timeseries> RedisTimeSeries version 80001, git_sha=577bfa8b5909e7ee572f0b651399be8303dc6641
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> Redis version found by RedisTimeSeries : 8.0.3 - oss
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> Registering configuration options: [
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> 	{ ts-compaction-policy   :              }
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> 	{ ts-num-threads         :            3 }
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> 	{ ts-retention-policy    :            0 }
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> 	{ ts-duplicate-policy    :        block }
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> 	{ ts-chunk-size-bytes    :         4096 }
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> 	{ ts-encoding            :   compressed }
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> 	{ ts-ignore-max-time-diff:            0 }
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> 	{ ts-ignore-max-val-diff :     0.000000 }
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> ]
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * <timeseries> Detected redis oss
redis-1          | 1:M 30 Jul 2025 01:46:15.709 * Module 'timeseries' loaded from /usr/local/lib/redis/modules//redistimeseries.so
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> Created new data type 'ReJSON-RL'
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> version: 80001 git sha: unknown branch: unknown
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> Exported RedisJSON_V1 API
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> Exported RedisJSON_V2 API
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> Exported RedisJSON_V3 API
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> Exported RedisJSON_V4 API
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> Exported RedisJSON_V5 API
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> Enabled diskless replication
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <ReJSON> Initialized shared string cache, thread safe: false.
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * Module 'ReJSON' loaded from /usr/local/lib/redis/modules//rejson.so
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * <search> Acquired RedisJSON_V5 API
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * Server initialized
redis-1          | 1:M 30 Jul 2025 01:46:15.711 * Ready to accept connections tcp
backend-2-1      | [Redis] Attempting connection...
backend-2-1      | [Redis] Connected successfully.
backend-1-1      | [Redis] Attempting connection...
backend-1-1      | [Redis] Connected successfully.
load-balancer-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
load-balancer-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
load-balancer-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
load-balancer-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
load-balancer-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
load-balancer-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
redis-1          | 1:signal-handler (1753840053) Received SIGTERM scheduling shutdown...
redis-1          | 1:M 30 Jul 2025 01:47:33.649 * User requested shutdown...
redis-1          | 1:M 30 Jul 2025 01:47:33.649 # Redis is now ready to exit, bye bye...
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
[Kload-balancer-1 exited with code 0
[Kredis-1 exited with code 0
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 9, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=1,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 8, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=2,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 7, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=3,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 6, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=4,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 5, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=5,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 4, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=6,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 3, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=7,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 2, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=8,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=9,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 0, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71720, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=10,CompletedItems=175456,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketClosed (WriteSocketError/ConnectionReset, last-recv: 0) on redis:6379/Interactive, Idle/MarkProcessed, last: PING, origin: ReadFromPipe, outstanding: 11, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |    at StackExchange.Redis.ServerEndPoint.<>c__DisplayClass43_0.<<OnConnectedAsync>g__IfConnectedAsync|0>d.MoveNext() + 0x12b
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.TaskExtensions.<TimeoutAfter>d__9.MoveNext() + 0xd0
backend-2-1      | --- End of stack trace from previous location ---
backend-2-1      |    at StackExchange.Redis.ConnectionMultiplexer.<WaitAllIgnoreErrorsAsync>d__89.MoveNext() + 0x320
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 9, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=0,CompletedItems=175212,Timers=6), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 8, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=0,CompletedItems=175212,Timers=6), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 7, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=2,CompletedItems=175212,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 6, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=3,CompletedItems=175212,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 5, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=4,CompletedItems=175212,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 4, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=5,CompletedItems=175212,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 3, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=6,CompletedItems=175212,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 1, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=8,CompletedItems=175212,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 0, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=9,CompletedItems=175212,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568, command=LPOP, timeout: 5000, inst: 0, qu: 2, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 6, async-ops: 71702, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=3,Free=32764,Min=1,Max=32767), POOL: (Threads=4,QueuedItems=7,CompletedItems=175212,Timers=7), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 50) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 10, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x32
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0x1e7
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x16e
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x57c
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__8.MoveNext() + 0x134
[Kbackend-2-1 exited with code 0
[Kbackend-1-1 exited with code 0
log truncated at line 1000
